releases/net472/KnownIssues/613745 - Single-name references are removed by the SDK when targeting 4.7.2.md

```diff
@@ -0,0 +1,23 @@
+# Single-name references are removed by the SDK when targeting 4.7.2 and referencing a package with the same assembly.
+
+## Symptoms
+When a project targets 4.7.2 and references an assembly with both a single name reference and a package reference (either a explicit or a transitive package reference), the assembly will not be passed as a reference to the compiler, causing build time errors for types that are not found. An example of a build error caused by this issue is:
+```
+error CS0246: The type or namespace name 'HttpClient' could not be found (are you missing a using directive or an assembly reference?)
+```
+
+Candidate assemblies that might run into this issue are: `System.Diagnostics.Tracing.dll`, `System.IO.Compression.dll`, and `System.Net.Http.dll`. Also is worth noting that this issue will only happen when the project uses `<PackageReference` model instead of `packages.config` to manage their NuGet dependencies.
+
+## Cause
+This issue will happen when a project has two references to the same assembly, one to the platform inbox implementation, and one to a package implementation. What ends up happening is that two different targets conflict with each other, and each one will remove one of the two references causing that when the call to the compiler is made, none of the references remain. NuGet targets will remove the Platform inbox implementation because it assumes that the package implementation should always be preferred. After this, the SDK targets will remove the package implementation because it assumes that all of the platform assemblies are referenced, and checks that the inbox assembly version is higher than the package one, so it prefers the one inbox.
```

Comment:
- will happen --> occurs
- platform inbox --> platform, or inbox platform. I'm not sure that inbox will be understood, though. It might be best to call this the platform implementation here and define what it is -- i.e., "one to the platform implementation (that is, the assembly that ships with the .NET Framework)")
- no comma after implementation on the first line.
a package implementation --> the package implementation
- two different targets --> the two different targets
- will remove --> removes (there are multiple occurrences of "will remove")
- references causing that... --> references, so that when the call to the compiler is made, none of the references remain
- Platform inbox implementation --> platform implementation (lowercase "p" in platform)
- it assumes --> they assume (2 occurrences)
- the final sentence is confusing. Perhaps, "The SDK targets remove the package implementation because they check that the platform assembly version is higher than the package one and, if it is, they assume the platform assembly is preferred.